Libraries/LibLine/XtermSuggestionDisplay.cpp

```diff
@@ -31,7 +31,7 @@
 
 namespace Line {
 
-void DumbTerminalSuggestionDisplay::display(const SuggestionManager& manager)
+void XtermSuggestionDisplay::display(const SuggestionManager& manager)
```

Comment:
This doesn't seem to belong in this commit.

Reply:
I thought I moved that change to its origin commit...ouch.

Reply:
Fixing that is gonna _hurt_...T_T

Reply:
Fixed :)